src/pem.rs

```diff
@@ -0,0 +1,78 @@
+#![allow(unused)]
+
+/// Split data by PEM guard lines
+pub struct PemBlock<'a> {
```

Comment:
Since this is only used by the schannel backend, could you move it into that module?